sdk/python/kfp_tekton/compiler/_op_to_template.py

```diff
@@ -451,16 +451,19 @@ def _op_to_template(op: BaseOp, enable_artifacts=False):
     if 'artifacts' in inputs:
         # The input artifacts in KFP is not pulling from s3, it will always be passed as a raw input.
         # Visit https://github.com/kubeflow/pipelines/issues/336 for more details on the implementation.
-        copy_inputs_step = _get_base_step('copy-inputs')
-        for artifact in inputs['artifacts']:
-            if 'raw' in artifact:
-                copy_inputs_step['script'] += 'echo -n "%s" > %s\n' % (artifact['raw']['data'], artifact['path'])
-            mount_path = artifact['path'].rsplit("/", 1)[0]
-            if mount_path not in mounted_param_paths:
-                _add_mount_path(artifact['name'], artifact['path'], mount_path,
-                                volume_mount_step_template, volume_template, mounted_param_paths)
-        template['spec']['steps'] = _prepend_steps([copy_inputs_step], template['spec']['steps'])
-        _update_volumes(template, volume_mount_step_template, volume_template)
+        # copy_inputs_step = _get_base_step('copy-inputs')
+        # for artifact in inputs['artifacts']:
+        #     if 'raw' in artifact:
+        #         copy_inputs_step['script'] += 'echo -n "%s" > %s\n' % (artifact['raw']['data'], artifact['path'])
+        #     mount_path = artifact['path'].rsplit("/", 1)[0]
+        #     if mount_path not in mounted_param_paths:
+        #         _add_mount_path(artifact['name'], artifact['path'], mount_path,
+        #                         volume_mount_step_template, volume_template, mounted_param_paths)
+        # template['spec']['steps'] = _prepend_steps([copy_inputs_step], template['spec']['steps'])
+        # _update_volumes(template, volume_mount_step_template, volume_template)
+        # Francis Todo: above info may need to be moved to bigger data passing logic
```

Comment:
Do we still need to keep these commented code since you already moved them to the bigger data passing logic?

Reply:
Oh, Thank you. Tomcli. I forgot delete it.